src/config.ts

```diff
@@ -36,6 +36,16 @@ const config: Config = {
   INTL_DIR,
   TRANSLATIONS_DIR: `${WORKING_DIR}/src/intl`,
   TRANSLATIONS_FILE: `${INTL_DIR}/english.source.json`,
+  DEEPL_ENGINE_ID: 418490,
+  DEEPL_SUPPORTED_LANGUAGES: [
+    "nl",
+    "en-GB",
+    "en-US",
+    "de",
+    "es-ES",
+    "fr",
+    "it",
+  ],
```

Comment:
These are different from `process.env.CROWDIN_LANGUAGES` as it wasn’t possible to enable `nl-BE` or `fr-BE` for DeepL.

Reply:
Move into `.env`.